TLM/TLM/UI/Helpers/ExtUITabStrip.cs

```diff
@@ -9,7 +9,7 @@ namespace TrafficManager.UI.Helpers {
 
     public sealed class ExtUITabstrip : UITabstrip {
 
-        public const float V_SCROLLBAR_WIDTH = 18f;
+        public const float V_SCROLLBAR_WIDTH = 16f;
```

Comment:
You're suffering because magic constants are used, and the game might possibly be sizing that scroll container differently based on screen resolution etc. Is it possible to derive it from container or contents width? Or does this change solve it once and for all.

Reply:
Naah, I just got a feeling that scrollbar was too wide :) 
AFAIK Game scales everything to reference resolution(FullHD), so everything is proportional on all resolutions. You can check Options Panel container height using diff game resolutions (it's `+-790f`) `ModTools` debug view will always show that value even if you run game in window `1024x768`